integration-tests/src/router.rs

```diff
@@ -152,13 +153,70 @@ mod router_different_pools_tests {
 			assert_balance!(BOB.into(), HDX, BOB_INITIAL_NATIVE_BALANCE);
 			assert_balance!(BOB.into(), DOT, amount_out);
 
-			expect_hydra_last_events(vec![pallet_route_executor::Event::Executed {
-				asset_in: DAI,
-				asset_out: DOT,
-				amount_in: amount_to_sell,
-				amount_out,
-			}
-			.into()]);
+			expect_hydra_events(vec![
+				pallet_amm_support::Event::Swapped {
+					swapper: BOB.into(),
+					filler: LBP::get_pair_id(pallet_lbp::types::AssetPair::new(DAI, LRNA)),
+					filler_type: pallet_amm_support::Filler::LBP,
+					operation: pallet_amm_support::TradeOperation::ExactIn,
+					inputs: vec![(AssetType::Fungible(DAI), 9980000000)],
+					outputs: vec![(AssetType::Fungible(LRNA), 5640664064)],
+					fees: vec![Fee::new(
+						DAI,
+						20000000,
+						LBP::pool_data(LBP::get_pair_id(pallet_lbp::types::AssetPair::new(DAI, LRNA)))
+							.unwrap()
+							.fee_collector,
+					)],
+					operation_id: vec![ExecutionType::Router(0)],
+				}
+				.into(),
+				pallet_amm_support::Event::Swapped {
+					swapper: BOB.into(),
```

Comment:
I guess we'll have 2 here. Also this taste is buying LRNA from LBP which is confusing as you can't do that.

Reply:
We're also minssing all of the fees here.

Reply:
yes, fixed, pls double check